index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Movie Site Project</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="/src/main.jsx"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/clapperboard.png" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Top 20 most popular movies</title>
```

Comment:
I did not thing that much of the title but when I saw yours I think it fits perfectly and adds the extra detail that it's the top 20. What I'm trying to say is that I like the title.